2021-09-17 18:37:37,535 INFO    MainThread:118293 [wandb_setup.py:_flush():69] setting env: {}
2021-09-17 18:37:37,536 INFO    MainThread:118293 [wandb_setup.py:_flush():69] setting login settings: {}
2021-09-17 18:37:37,536 INFO    MainThread:118293 [wandb_init.py:_log_setup():342] Logging user logs to /home/indika/Programming/Projects/Python/Artifical-Intelligence/PyTorch/CNN/Intel-Image-Classification-TL/wandb/run-20210917_183737-3ln5enka/logs/debug.log
2021-09-17 18:37:37,536 INFO    MainThread:118293 [wandb_init.py:_log_setup():343] Logging internal logs to /home/indika/Programming/Projects/Python/Artifical-Intelligence/PyTorch/CNN/Intel-Image-Classification-TL/wandb/run-20210917_183737-3ln5enka/logs/debug-internal.log
2021-09-17 18:37:37,536 INFO    MainThread:118293 [wandb_init.py:init():375] calling init triggers
2021-09-17 18:37:37,536 INFO    MainThread:118293 [wandb_init.py:init():382] wandb.init called with sweep_config: {}
config: {}
2021-09-17 18:37:37,536 INFO    MainThread:118293 [wandb_init.py:init():397] re-initializing run, found existing run on stack: 103r8z3i
2021-09-17 18:37:37,537 INFO    MainThread:118293 [wandb_run.py:finish():1271] finishing run ranuga-d/Intel-Image-Classification-TL/103r8z3i
2021-09-17 18:37:37,541 INFO    MainThread:118293 [jupyter.py:save_history():382] saving 13 cells to _session_history.ipynb
2021-09-17 18:37:37,545 INFO    MainThread:118293 [wandb_run.py:_config_callback():843] config_cb ('_wandb', 'session_history') code/_session_history.ipynb None
2021-09-17 18:37:37,553 INFO    MainThread:118293 [jupyter.py:save_ipynb():301] looking for notebook: 00.ipynb
2021-09-17 18:37:38,023 INFO    MainThread:118293 [wandb_init.py:_jupyter_teardown():274] saved code and history
2021-09-17 18:37:38,023 INFO    MainThread:118293 [wandb_init.py:_jupyter_teardown():275] cleaning up jupyter logic
2021-09-17 18:37:38,024 INFO    MainThread:118293 [wandb_run.py:_atexit_cleanup():1571] got exitcode: 0
2021-09-17 18:37:38,024 INFO    MainThread:118293 [wandb_run.py:_restore():1543] restore
2021-09-17 18:37:43,331 INFO    MainThread:118293 [wandb_run.py:_wait_for_finish():1693] got exit ret: None
2021-09-17 18:37:47,457 INFO    MainThread:118293 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 1
  artifact_count: 3
}
pusher_stats {
  uploaded_bytes: 36953
  total_bytes: 36953
  deduped_bytes: 16944
}

2021-09-17 18:37:48,072 INFO    MainThread:118293 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 1
  artifact_count: 3
}
pusher_stats {
  uploaded_bytes: 36953
  total_bytes: 36953
  deduped_bytes: 16944
}

2021-09-17 18:37:48,828 INFO    MainThread:118293 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 6
  artifact_count: 3
  other_count: 1
}
pusher_stats {
  uploaded_bytes: 36953
  total_bytes: 73402
  deduped_bytes: 16944
}

2021-09-17 18:37:48,931 INFO    MainThread:118293 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 6
  artifact_count: 3
  other_count: 1
}
pusher_stats {
  uploaded_bytes: 36953
  total_bytes: 73402
  deduped_bytes: 16944
}

2021-09-17 18:37:49,032 INFO    MainThread:118293 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 6
  artifact_count: 3
  other_count: 1
}
pusher_stats {
  uploaded_bytes: 36953
  total_bytes: 73402
  deduped_bytes: 16944
}

2021-09-17 18:37:49,134 INFO    MainThread:118293 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 6
  artifact_count: 3
  other_count: 1
}
pusher_stats {
  uploaded_bytes: 36953
  total_bytes: 73402
  deduped_bytes: 16944
}

2021-09-17 18:37:49,236 INFO    MainThread:118293 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 6
  artifact_count: 3
  other_count: 1
}
pusher_stats {
  uploaded_bytes: 36953
  total_bytes: 73402
  deduped_bytes: 16944
}

2021-09-17 18:37:49,337 INFO    MainThread:118293 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 6
  artifact_count: 3
  other_count: 1
}
pusher_stats {
  uploaded_bytes: 36953
  total_bytes: 73402
  deduped_bytes: 16944
}

2021-09-17 18:37:49,439 INFO    MainThread:118293 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 6
  artifact_count: 3
  other_count: 1
}
pusher_stats {
  uploaded_bytes: 36953
  total_bytes: 73402
  deduped_bytes: 16944
}

2021-09-17 18:37:49,540 INFO    MainThread:118293 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 6
  artifact_count: 3
  other_count: 1
}
pusher_stats {
  uploaded_bytes: 73402
  total_bytes: 73402
  deduped_bytes: 16944
}

2021-09-17 18:37:49,643 INFO    MainThread:118293 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 6
  artifact_count: 3
  other_count: 1
}
pusher_stats {
  uploaded_bytes: 73402
  total_bytes: 73402
  deduped_bytes: 16944
}

2021-09-17 18:37:49,745 INFO    MainThread:118293 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 6
  artifact_count: 3
  other_count: 1
}
pusher_stats {
  uploaded_bytes: 73402
  total_bytes: 73402
  deduped_bytes: 16944
}

2021-09-17 18:37:49,847 INFO    MainThread:118293 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 6
  artifact_count: 3
  other_count: 1
}
pusher_stats {
  uploaded_bytes: 73402
  total_bytes: 73402
  deduped_bytes: 16944
}

2021-09-17 18:37:49,949 INFO    MainThread:118293 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 6
  artifact_count: 3
  other_count: 1
}
pusher_stats {
  uploaded_bytes: 73402
  total_bytes: 73402
  deduped_bytes: 16944
}

2021-09-17 18:37:50,050 INFO    MainThread:118293 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 6
  artifact_count: 3
  other_count: 1
}
pusher_stats {
  uploaded_bytes: 73402
  total_bytes: 73402
  deduped_bytes: 16944
}

2021-09-17 18:37:50,152 INFO    MainThread:118293 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 6
  artifact_count: 3
  other_count: 1
}
pusher_stats {
  uploaded_bytes: 73402
  total_bytes: 73402
  deduped_bytes: 16944
}

2021-09-17 18:37:50,254 INFO    MainThread:118293 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 6
  artifact_count: 3
  other_count: 1
}
pusher_stats {
  uploaded_bytes: 73402
  total_bytes: 73402
  deduped_bytes: 16944
}

2021-09-17 18:37:50,622 INFO    MainThread:118293 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 6
  artifact_count: 3
  other_count: 1
}
pusher_stats {
  uploaded_bytes: 73402
  total_bytes: 73402
  deduped_bytes: 16944
}

2021-09-17 18:37:50,724 INFO    MainThread:118293 [wandb_run.py:_wait_for_finish():1693] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 6
  artifact_count: 3
  other_count: 1
}
pusher_stats {
  uploaded_bytes: 73402
  total_bytes: 73402
  deduped_bytes: 16944
}

2021-09-17 18:37:51,823 INFO    MainThread:118293 [wandb_run.py:_show_files():1915] logging synced files
2021-09-17 18:37:51,828 INFO    MainThread:118293 [wandb_init.py:init():424] starting backend
2021-09-17 18:37:51,828 INFO    MainThread:118293 [backend.py:_multiprocessing_setup():72] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2021-09-17 18:37:51,831 INFO    MainThread:118293 [backend.py:ensure_launched():135] starting backend process...
2021-09-17 18:37:51,864 INFO    MainThread:118293 [backend.py:ensure_launched():140] started backend process with pid: 129658
2021-09-17 18:37:51,865 INFO    MainThread:118293 [wandb_init.py:init():429] backend started and connected
2021-09-17 18:37:51,876 INFO    MainThread:118293 [wandb_run.py:_label_probe_notebook():815] probe notebook
2021-09-17 18:37:51,877 INFO    MainThread:118293 [wandb_init.py:init():477] updated telemetry
2021-09-17 18:37:51,877 INFO    MainThread:118293 [wandb_init.py:init():500] communicating current version
2021-09-17 18:37:52,770 INFO    MainThread:118293 [wandb_init.py:init():505] got version response upgrade_message: "wandb version 0.12.2 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2021-09-17 18:37:52,770 INFO    MainThread:118293 [wandb_init.py:init():513] communicating run to backend with 30 second timeout
2021-09-17 18:37:53,266 INFO    MainThread:118293 [wandb_init.py:init():540] starting run threads in backend
2021-09-17 18:37:58,273 INFO    MainThread:118293 [wandb_run.py:_console_start():1601] atexit reg
2021-09-17 18:37:58,274 INFO    MainThread:118293 [wandb_run.py:_redirect():1475] redirect: SettingsConsole.WRAP
2021-09-17 18:37:58,274 INFO    MainThread:118293 [wandb_run.py:_redirect():1512] Wrapping output streams.
2021-09-17 18:37:58,275 INFO    MainThread:118293 [wandb_run.py:_redirect():1536] Redirects installed.
2021-09-17 18:37:58,275 INFO    MainThread:118293 [wandb_init.py:init():565] run started, returning control to user process
2021-09-17 18:38:15,982 INFO    MainThread:118293 [wandb_setup.py:_flush():69] setting env: {}
2021-09-17 18:38:15,982 INFO    MainThread:118293 [wandb_setup.py:_flush():69] setting login settings: {}
2021-09-17 18:38:15,982 INFO    MainThread:118293 [wandb_init.py:_log_setup():342] Logging user logs to /home/indika/Programming/Projects/Python/Artifical-Intelligence/PyTorch/CNN/Intel-Image-Classification-TL/wandb/run-20210917_183815-22xpfvmx/logs/debug.log
2021-09-17 18:38:15,982 INFO    MainThread:118293 [wandb_init.py:_log_setup():343] Logging internal logs to /home/indika/Programming/Projects/Python/Artifical-Intelligence/PyTorch/CNN/Intel-Image-Classification-TL/wandb/run-20210917_183815-22xpfvmx/logs/debug-internal.log
2021-09-17 18:38:15,983 INFO    MainThread:118293 [wandb_init.py:_jupyter_setup():294] configuring jupyter hooks <wandb.sdk.wandb_init._WandbInit object at 0x7f4999d04150>
2021-09-17 18:38:15,983 INFO    MainThread:118293 [wandb_init.py:init():375] calling init triggers
2021-09-17 18:38:15,983 INFO    MainThread:118293 [wandb_init.py:init():382] wandb.init called with sweep_config: {}
config: {}
2021-09-17 18:38:15,983 INFO    MainThread:118293 [wandb_init.py:init():397] re-initializing run, found existing run on stack: 3ln5enka
2021-09-17 18:38:15,984 INFO    MainThread:118293 [wandb_run.py:finish():1271] finishing run ranuga-d/Intel-Image-Classification-TL/3ln5enka
